ts/compile/builder/crates/core/src/main.rs

```diff
@@ -54,8 +54,8 @@ fn nextwrap(context: *mut JSContext, _jsval1: JSValue, _int1: c_int, jsval2: *mu
     let ptr = JS_GetPropertyUint32(context, *jsval2, 0);    
     let len = JS_GetPropertyUint32(context, *jsval2, 1);    
 
-    let packed = _next(ptr as u32, len as u32);
-    return JS_NewInt64_Ext(context, packed as i64);    
+    _next(ptr as u32, len as u32);
+    return JS_NewInt64_Ext(context, 0);   // Dummy, not really required.
```

Comment:
If it's not required then should we remove the argument?

Reply:
Fixed to return undefined instead of 0.
68661152f9eaa557e7797cabb1d215c126a0f26f